src/dotty/tools/dotc/typer/Typer.scala

```diff
@@ -166,9 +171,9 @@ class Typer extends Namer with TypeAssigner with Applications with Implicits wit
         else {
           if (!scala2pkg && !previous.isError && !found.isError) {
             error(
-              ex"""reference to $name is ambiguous;
-                  |it is both ${bindingString(newPrec, ctx, "")}
-                  |and ${bindingString(prevPrec, prevCtx, " subsequently")}""",
+              ex"""|reference to `$name` is ambiguous
```

Comment:
Since we're applying `stripMargin` to the result of the string interpolation, the result is the same with or without the first `|`.

I simply like it present sine it can then be aligned with the other pipe symbols showing clearly what the resulting string will look like.